third_party/types/common/callback.go

```diff
@@ -13,10 +15,16 @@ func DefaultCallback() MotorCallback {
 	return &defaultCallback{}
 }
 
-// func (cb *defaultCallback) OnMotorEvent(msg string, isDone bool) {
-// 	log.Printf("message: %s done: %t", msg, isDone)
-// }
-
 func (cb *defaultCallback) OnDiscover(data []byte) {
-	log.Println("ERROR: MotorCallback not implemented.")
+	log.Println(ErrDefaultStillImplemented.Error())
+}
+
+func (cb *defaultCallback) OnWalletEvent(data []byte) {
```

Comment:
Can these changes be unit tested?

Reply:
we can unit test the implementation of the interface. and use the utility methods